toplevel/lib/dune

```diff
@@ -4,5 +4,6 @@
   (synopsis "Js_of_ocaml toplevel library")
   (libraries
     js_of_ocaml-compiler js_of_ocaml bytes
-    compiler-libs.bytecomp compiler-libs.toplevel)
+    compiler-libs.bytecomp compiler-libs.toplevel
+    ppxlib)
```

Comment:
Why do we have an explicit dependency on ppxlib here ?

Reply:
That's not needed. Thanks for noticing! I've removed it.

By the way, I also explicitly increased the compiler version dependency bound to `>= "4.04.0"` (from `>= "4.02.0"`) both in `js_of_ocaml-ppx.opam` and `js_of_ocaml.opam`, which is also not needed. Do you maybe also want me to revert that?

Reply:
If it's not needed, yes please

Reply:
What I meant was that 4.04.0 is just an implicit bound due to using ppxlib, but I didn't need to make it explicit. I've gone back to 4.02.0.